package.json

```diff
@@ -89,5 +89,8 @@
     "plugins": [
       "preset-wooorm"
     ]
+  },
+  "engines": {
+    "node": ">=8"
```

Comment:
I don't think we officially support 9 or 11, which are no longer supported.
https://github.com/nodejs/Release

Maybe `^8.0.0 || ^10.0.0 || >= 12.0.0` would be more accurate.

Reply:
I’m personally in favour of not having `engines` at all. I found that they are more often erroneous, while maintaining projects like this for years, then useful.

Reply:
I figure it's the ideal way to signal to consumers what min node version you support, but I can take it out.